docs/configuration.md

```diff
@@ -1600,6 +1600,33 @@ Apart from these, the following properties are also available, and may be useful
     How many batches the Spark Streaming UI and status APIs remember before garbage collecting.
   </td>
 </tr>
+<tr>
+  <td><code>spark.streaming.driver.writeAheadLog.closeFileAfterWrite</code></td>
+  <td>false</td>
+  <td>
+    Whether to close the file after writing a write ahead log record in driver. Because S3 doesn't
+    support flushing of data, when using S3 for checkpointing, you should enable it to achieve read
+    after write consistency.
+  </td>
+</tr>
+<tr>
+  <td><code>spark.streaming.receiver.writeAheadLog.closeFileAfterWrite</code></td>
+  <td>false</td>
+  <td>
+    Whether to close the file after writing a write ahead log record in receivers. Because S3
+    doesn't support flushing of data, when using S3 for checkpointing, you should enable it to
+    achieve read after write consistency.
+  </td>
+</tr>
+<tr>
+  <td><code>spark.streaming.driver.writeAheadLog.allowBatching</code></td>
+  <td>false</td>
```

Comment:
for me: the default value is `true`.

That's why I want to expose this one since the behavior is different from 1.5.0.